libpod/container.go

```diff
@@ -293,6 +293,10 @@ type ContainerConfig struct {
 	// namespace
 	// These are not used unless CreateNetNS is true
 	PortMappings []ocicni.PortMapping `json:"portMappings,omitempty"`
+	// UseImageResolvConf indicates that resolv.conf should not be
```

Comment:
does "should not" still hold or should 'not' be dropped?

Reply:
I think this is still accurate. The description is still negative, not positive, but it does describe what the flag does.